src/main/java/com/google/cloud/tools/gradle/appengine/core/ManagedCloudSdkFactory.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2018 Google Inc. All Right Reserved.
```

Comment:
we tend not to use (c) in our copyrights and arguably it's not correct legal form: https://g3doc.corp.google.com/company/teams/opensource/copyright.md?cl=head

Reply:
I realize I told you to do this wholesale on the project, but I think protocol is to fix this in this PR and then fix the non-PR related files in a followup.